### PR TITLE
replace *_filter with *_action

### PR DESCRIPTION
**Pull request by BenMorganIO**
This PR will help with getting Ahoy Rails 5 ready :)

```
DEPRECATION WARNING: before_filter is deprecated and will be removed in Rails 5.1. Use before_action instead.
```

### CODE REVIEW

File: app/controllers/ahoy/base_controller.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module Ahoy
   class BaseController < ApplicationController
-    # skip all filters except for authlogic
-    skip_filter *(_process_action_callbacks.map(&:filter) - [:load_authlogic])
+    # skip all actions except for authlogic
+    skip_action(*(_process_action_callbacks.map(&:filter) - [:load_authlogic]))
```
Comment (BenMorganIO) [inline app/controllers/ahoy/base_controller.rb]: Not sure if this should be `.map(&:action)`.

**Comment by smasry** [comment]
+1

**Comment by jurajmasar** [comment]
+1

**Comment by ankane** [comment]
Thanks Ben! To preserve backward compatibility, went with 519a6e6806c543509097399a9cdfa3acd9a0225a and 224a6397f8a292c48987015cbe22096780536a9f

**Comment by BenMorganIO** [comment]
@ankane I'm pretty sure `before_filter` as an alias of `before_action` :/

**Comment by ankane** [comment]
Unfortunately, this PR as is breaks compatibility with Rails 3.2.

**Pull request closed without merging**